### PR TITLE
preparation for 7.0.0-beta3 release 

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| new release
|New feature| No 
|BC breaks| No 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/migrations/composer.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7|^6.0|^7.0",
-        "shopsys/coding-standards": "dev-master"
+        "shopsys/coding-standards": "7.0.0-beta3"
```
Comment (boris-brtan) [inline packages/migrations/composer.json]: oooOOOoooo so basic mistake. It should begin with `v` not `number`

Comment (boris-brtan) [reply packages/migrations/composer.json]: sorrry my mistake

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,80 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
+
+#### Changed
+
+- [#576] `OrderFormType` in administration is now rendered by default
+
+#### Removed
+
+- [#595] removal of automatic price calculation
+
+#### Fixed
```
Comment (boris-brtan) [inline CHANGELOG.md]: fixed is somehow after removed and changed although `ch` is not the same as `c` `h` in english language

File: CHANGELOG.md [hunk 3]
```diff
@@ -1644,3 +1718,64 @@ That's why is this section formatted differently.
 [#487]: https://github.com/shopsys/shopsys/pull/487
 [#486]: https://github.com/shopsys/shopsys/pull/486
 [@thirdknown]: https://github.com/thirdknown
+
```
Comment (boris-brtan) [inline CHANGELOG.md]: i am not sure whether this new line feed makes sense

**Comment by boris-brtan** [comment]
TESTS: force split on travis seems OK, kubernetes CI with builded microservices seems OK

File: packages/framework/composer.json [hunk 4]
```diff
@@ -68,12 +68,12 @@
...
+        "shopsys/doctrine-orm": "^2.6.3",
         "shopsys/jparser": "0.1",
         "shopsys/postgres-search-bundle": "0.1",
-        "shopsys/migrations": "dev-master",
-        "shopsys/form-types-bundle": "dev-master",
-        "shopsys/plugin-interface": "dev-master",
+        "shopsys/migrations": "7.0.0-beta3",
+        "shopsys/form-types-bundle": "7.0.0-beta3",
+        "shopsys/plugin-interface": "7.0.0-beta3",
         "snc/redis-bundle": "2.1.4",
```
Comment (boris-brtan) [inline packages/framework/composer.json]: very strange, not event human factor but also computer factor forgets to fix this package with `^` notation https://github.com/shopsys/shopsys/issues/549

File: project-base/composer.json [hunk 5]
```diff
@@ -66,16 +66,16 @@
...
-        "shopsys/product-feed-google": "dev-master",
+        "shopsys/migrations": "7.0.0-beta3",
+        "shopsys/form-types-bundle": "7.0.0-beta3",
+        "shopsys/framework": "7.0.0-beta3",
+        "shopsys/plugin-interface": "7.0.0-beta3",
+        "shopsys/product-feed-heureka": "7.0.0-beta3",
+        "shopsys/product-feed-heureka-delivery": "7.0.0-beta3",
+        "shopsys/product-feed-zbozi": "7.0.0-beta3",
+        "shopsys/product-feed-google": "7.0.0-beta3",
         "snc/redis-bundle": "2.1.4",
```
Comment (boris-brtan) [inline project-base/composer.json]: ^

File: CHANGELOG.md [hunk 6]
```diff
@@ -28,6 +28,80 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## v7.0.0-beta3 - 2018-11-26
```
[the remark this answers is not in the document]
Comment (vitek-rostislav) [reply CHANGELOG.md]: I know, the date check is a part of releasing process after review :slightly_smiling_face:

**Pull request merged**